AI Kit Product name change (#2013)
oneapi-src/oneAPI-samples: AI-and-Analytics/Features-and-Functionality/README.md

# Features and Functionalities for AI Tools

The Intel AI Tools give data scientists, AI developers, and researchers familiar Python* tools and frameworks to accelerate end-to-end data science and analytics pipelines on Intel® architectures. The components are built using oneAPI libraries for low-level compute optimizations. This toolkit maximizes performance from preprocessing through machine learning, and provides interoperability for efficient model development.

You can find more information at [ AI Tools](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html).

> Users could learn more details of features in AI Tools with those features and functionality samples.

Reviewer: ```suggestion
Users can explore the extensive features of Intel AI Tools through provided feature and functionality samples, offering a deeper understanding of their capabilities.
```

| TensorFlow & Reference Models | [IntelTensorFlow Reference Models Inference with FP32 Int8](IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8)               | Run ResNet50 inference on Intel's pretrained FP32 and Int8 model.
| TensorFlow | [IntelTensorFlow PerformanceAnalysis](IntelTensorFlow_PerformanceAnalysis) | Analyze the performance difference between Stock Tensorflow and Intel Tensorflow.
| TensorFlow | [IntelTensorFlow Transformer AMX bfloat16 MixedPrecisiong](IntelTensorFlow_Transformer_AMX_bfloat16_MixedPrecision) | Run a transformer classification model with bfloat16 mixed precision.
| TensorFlow | [IntelTensorFlow for LLMs](IntelTensorFlow_for_LLMs) | Finetune a GPT-J (LLM) model using the GLUE cola dataset with the Intel® Optimization for TensorFlow*.
| Scikit-learn | [IntelScikitLearn Extensions SVC Adult](IntelScikitLearn_Extensions_SVC_Adult)   | Use Intel® Extension for Scikit-learn to accelerate the training and prediction with SVC algorithm on Adult dataset. Compare the performance of SVC algorithm optimized through Intel® Extension for Scikit-learn against original Scikit-learn..

> 

Reviewer 2: At the end of the file we need to insert:
*Other names and brands may be claimed as the property of others. [Trademarks](http://www.intel.com/content/www/us/en/legal/trademarks.html)

This request is relevant for any README file where we use asterisks

Author: addressed it accordingly.

| Compoment      | Folder                                             | Description
| --------- | ------------------------------------------------ | -

> | Intel® Neural Compressor | [INC Quantization Aware Training](INC_QuantizationAwareTraining_TextClassification)                     | fine-tune BERT tiny model for emotion classification task using Quantization Aware Training (QAT) from Intel® Neural Compressor.

Reviewer: Please apply the suggested changes in samples description as well

.
| Numpy, Numba | [IntelPython Numpy Numba dpex kNN](IntelPython_Numpy_Numba_dpex_kNN)   | Optimize k-NN model by numba_dpex operations without sacrificing accuracy.
| XGBoost | [IntelPython XGBoost Performance](IntelPython_XGBoost_Performance) |  Analyze the performance benefit from using Intel optimized XGBoost compared to un-optimized XGBoost 0.81.
| XGBoost | [IntelPython XGBoost daal4pyPrediction](IntelPython_XGBoost_daal4pyPrediction) |  Analyze the performance benefit of minimal code changes to port pre-trained XGBoost model to daal4py prediction for much faster prediction than XGBoost prediction.
| daal4py | [IntelPython daal4py DistributedKMeans](IntelPython_daal4py_DistributedKMeans)    | train and predict with a distributed k-means model using the python API package daal4py powered by the oneAPI Data Analytics Library.
| daal4py | [IntelPython daal4py DistributedLinearRegression](IntelPython_daal4py_DistributedLinearRegression)    | Run a distributed Linear Regression model with oneAPI Data Analytics Library (oneDAL) daal4py library memory objects.

> | PyTorch | [IntelPytorch Interactive Chat Quantization](IntelPytorch_Interactive_Chat_Quantization)   | Create interactive chat based on pre-trained DialoGPT model and add the Intel® Extension for PyTorch* quantization to it.

Reviewer: ```suggestion
| Intel® Optimization for PyTorch* | [IntelPytorch Interactive Chat Quantization](IntelPytorch_Interactive_Chat_Quantization)   | Create an interactive chat based on a pre-trained DialoGPT model and add the Intel® Optimization for PyTorch* quantization to it.
```

> | PyTorch | [IntelPytorch Quantization](IntelPytorch_Quantization)   | Inference performance improvements using Intel® Extension for PyTorch* (IPEX) with feature quantization.

Reviewer: ```suggestion
| Intel® Optimization for PyTorch* | [IntelPytorch Quantization](IntelPytorch_Quantization)   | Gain inference performance improvements using Intel® Optimization for PyTorch* with feature quantization.
```

> | TensorFlow | [IntelTensorFlow AMX BF16 Training](IntelTensorFlow_AMX_BF16_Inference) | Enabling auto-mixed precision to use low-precision datatypes, like bfloat16, for model inference with TensorFlow* .

Reviewer: ```suggestion
| Intel® Optimization for TensorFlow* | [IntelTensorFlow AMX BF16 Training](IntelTensorFlow_AMX_BF16_Inference) | Enable auto-mixed precision to use low-precision datatypes, like bfloat16, for model inference with TensorFlow*.
```

> | TensorFlow | [IntelTensorFlow AMX BF16 Training](IntelTensorFlow_AMX_BF16_Training) | Training performance improvements with Intel® AMX BF16.

Reviewer: ```suggestion
| Intel® Optimization for TensorFlow* | [IntelTensorFlow AMX BF16 Training](IntelTensorFlow_AMX_BF16_Training) | Gain training performance improvements with Intel® AMX Bfloat16.
```

| TensorFlow | [IntelTensorFlow Enabling Auto Mixed Precision for TransferLearning](IntelTensorFlow_Enabling_Auto_Mixed_Precision_for_TransferLearning) | Enabling auto-mixed precision to use low-precision datatypes, like bfloat16, for transfer learning with TensorFlow*.

> | Horovod | [IntelTensorFlow Horovod Distributed Deep Learning](IntelTensorFlow_Horovod_Distributed_Deep_Learning) | run inference & training workloads across multi-cards using Intel Optimization for Horovod and TensorFlow* on Intel® dGPU's.

Reviewer: ```suggestion
| Horovod | [IntelTensorFlow Horovod Distributed Deep Learning](IntelTensorFlow_Horovod_Distributed_Deep_Learning) | Run inference & training workloads across multi-cards using Intel Optimization for Horovod and TensorFlow* on Intel® dGPU's.
```

Reviewer: and Intel Optimization for TensorFlow*?

> | TensorFlow | [IntelTensorFlow InferenceOptimization](IntelTensorFlow_InferenceOptimization) |  Optimize a pre-trained model for a better inference performance.

Reviewer: ```suggestion
| Intel® Optimization for TensorFlow | [IntelTensorFlow InferenceOptimization](IntelTensorFlow_InferenceOptimization) |  Optimize a pre-trained model for a better inference performance.
```

> | TensorFlow & Reference Models | [IntelTensorFlow Reference Models Inference with FP32 Int8](IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8)               | Run ResNet50 inference on Intel's pretrained FP32 and Int8 model.

Reviewer: ```suggestion
| Intel® Optimization for TensorFlow & Intel AI Reference Models | [IntelTensorFlow Reference Models Inference with FP32 Int8](IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8)               | Run ResNet50 inference on Intel's pretrained FP32 and Int8 model.
```

> | TensorFlow | [IntelTensorFlow PerformanceAnalysis](IntelTensorFlow_PerformanceAnalysis) | Analyze the performance difference between Stock Tensorflow and Intel Tensorflow.

Reviewer: ```suggestion
| Intel® Extension for TensorFlow* | [IntelTensorFlow PerformanceAnalysis](IntelTensorFlow_PerformanceAnalysis) | Analyze the performance difference between Stock Tensorflow and Intel Tensorflow.
```

Author: no ITEX involved here.

> | TensorFlow | [IntelTensorFlow Transformer AMX bfloat16 MixedPrecisiong](IntelTensorFlow_Transformer_AMX_bfloat16_MixedPrecision) | Run a transformer classification model with bfloat16 mixed precision.

Reviewer: Intel® Optimization for TensorFlow

> | TensorFlow | [IntelTensorFlow for LLMs](IntelTensorFlow_for_LLMs) | Finetune a GPT-J (LLM) model using the GLUE cola dataset with the Intel® Optimization for TensorFlow*.

Reviewer: ```suggestion
| Intel® Optimization for TensorFlow| [IntelTensorFlow for LLMs](IntelTensorFlow_for_LLMs) | Finetune a GPT-J (LLM) model using the GLUE cola dataset with the Intel® Optimization for TensorFlow*.
```
